src/pages/teams.astro

```diff
@@ -0,0 +1,363 @@
+            collaboration</span
+          >
+        </div>
+        <hr class="border-gray-800" />
+      </div>
+    </div>
+  </section>
+  <section id="newsletter" class="bg-gray-900">
+    <div class="m-auto max-w-6xl px-4 py-6">
+      <div class="flex items-center justify-between gap-10">
+        <img src={newsletter} class="hidden sm:block w-[50%]" />
+        <div class="flex flex-col gap-5">
+          <h2
+            class="text-gray-50 text-2xl sm:text-4xl leading-10 text-center sm:text-left"
+          >
+            Stay Updated on <br class="hidden sm:block" /> Livebook Teams' Progress
+          </h2>
+          <span class="text-gray-200"
+            >Interested in Livebook Teams but not quite ready to create a beta
+            account? No problem! Sign up for our newsletter to receive updates
+            on the latest features, improvements, and news about Livebook Teams.</span
+          >
+
+          <form
+            action="URL"
+            method="post"
+            class="flex flex-col sm:flex-row items-center gap-4"
+          >
+            <input
+              type="email"
+              placeholder="Email Adress"
+              class="block w-full px-3 py-2 bg-gray-900 border border-gray-600 rounded-md text-sm text-gray-200 shadow-sm placeholder-gray-400 focus:outline-none focus:border-blue-500 focus:ring-1 focus:ring-blue-500 disabled:bg-slate-50 disabled:text-gray-500 disabled:border-gray-200 disabled:shadow-none invalid:border-brand-pink invalid:text-brand-pink focus:invalid:border-brand-pink focus:invalid:ring-brand-pink"
+            />
+            <button type="submit" class="button button-blue">
+              <span>Subscribe</span>
+            </button>
+          </form>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section id="get-in-touch" class="bg-gray-900">
+    <div class="px-4 pt-12 pb-20">
+      <div
+        class="m-auto max-w-[1120px] bg-gradient-to-r from-brand-pink to-blue-600 rounded-[32px] text-center p-[2px]"
+      >
+        <div class="h-full w-full rounded-[32px] bg-gray-900 py-12 lg:py-20">
+          <div class="m-auto max-w-2xl flex flex-col items-center">
+            <Icon icon="question-answer-fill text-3xl text-[#e44c75]" />
+            <h2 class="text-gray-50 text-2xl sm:text-3xl leading-tight mt-2">
+              Have questions about <br class="hidden sm:block" /> Livebook Teams?
+            </h2>
+            <span class="lg:w-[70%] text-gray-200 mt-6 px-2 sm:px-0"
+              >If you have any questions about Livebook Teams, we're here to
+              help. Don't hesitate to contact our team; we'll gladly provide the
+              information you need.</span
+            >
+            <a class="button button-gray mt-8">
+              <span> Send an Email</span>
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</Layout>
+
+<script>
+  document.addEventListener("DOMContentLoaded", function () {
```

Comment:
We don't need to wrap it in `DOMContentLoaded`, the script is imported as `type="module"` and it always runs after the whole page is rendered.